scanner/src/main/java/ca/on/oicr/gsi/runscanner/scanner/RestResponseController.java

```diff
@@ -63,6 +63,24 @@ public ResponseEntity<NotificationDto> getByName(
         .orElseGet(() -> new ResponseEntity<>(HttpStatus.NOT_FOUND));
   }
 
+  @DeleteMapping("/run/{name}")
+  @ApiOperation(
+      value = "Invalidate cache for run by name",
+      response = Void.class,
+      responseContainer = "ResponseEntity")
+  @ApiResponses({
+    @ApiResponse(code = 204, message = "Success"),
+    @ApiResponse(code = 404, message = "Run not found")
+  })
+  public ResponseEntity<Void> deleteByName(
+      @PathVariable("name") @ApiParam(value = "Run name") String id) {
+    if (scheduler.invalidate(id)) {
+      return new ResponseEntity<>(HttpStatus.NO_CONTENT);
+    } else {
+      return new ResponseEntity<>(HttpStatus.NOT_FOUND);
+    }
+  }
```

Comment:
You can make this simpler by removing the `ResponseEntity` and just making the method return `void`. Throw `ResourceNotFoundException` if the run isn't found. I'm guessing for Swagger, you'd just remove the `response` and `responseContainer` parameters since it's not returning anything

Reply:
You'd have to add Spring's `@ResponseStatus` annotation to the method to make it return 204 with this change

Reply:
Fixed, thanks!